server/users/service.go

```diff
@@ -242,9 +265,10 @@ func (rs *RecognitionService) runPipeline(destinationID string, identifiers Even
 			continue
 		}
 
-		err = configuration.UserIDJSONPath.Set(event, identifiers.UserID)
+		err = configuration.PropertyJSONPathes.Set(event, identifiers.Properties)
```

Comment:
Using JSON paths array with values array is not reliable. What happens when the order of `identification_nodes` is changed (e.g. by a user or by JSON serializer in the case when the server loads destinations configuration from URL source)? Wrong values will be set and all recognized events will contain an error. Let's think about how to avoid this situation? (e.g. by using hashmap)

Reply:
Rewritten to use map